.github/workflows/publish_fgpyo.yml

```
@@ -0,0 +1,134 @@
+name: publish fgpyo
+
+on:
+  push:
+    tags: '\d+.\d+.\d+'
```

actionlint (1.7.4)

character '\' is invalid for branch and tag names. only special characters [, ?, +, *, \, ! can be escaped with \. see `man git-check-ref-format` for more details. note that regular expression is unavailable. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)